reflex-dom-core/src/Reflex/Dom/Main.hs

```diff
@@ -15,6 +15,8 @@ module Reflex.Dom.Main where
 
 import Prelude hiding (concat, mapM, mapM_, sequence, sequence_)
 
+import Reflex.Class (holdDyn)
+import Reflex.Dom.Builder.Class (MountState(Mounted, Mounting))
```

Comment:
corrected. I couldn't get stylish-haskell to run in a way convenient to check its complaints as it seems to want to be a pass-through filter only? running it from the root just had it hanging waiting for stdin.